docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
+
+When things escalate, try to use this document as a guideline on how to de-escalate them.
```

Comment:
Changing this to "how to diffuse" would sound a bit clearer. Reusing the same word in a hyphenated negation tends to make it sound like there is not much clarity that can be offered about the word.